src/explore-education-statistics-admin/src/services/releaseService.ts

```diff
@@ -122,14 +122,17 @@ export interface ReleaseStageStatuses {
   lastUpdated?: string;
 }
 
+export const ReleaseChecklistErrorCode = [
```

Comment:
Is there any way to use this in ReleaseStatusChecklist to prevent them becoming out of sync if it changes?

Reply:
In part, yes!

In ReleaseStatusChecklist, the error codes in its switch statement are confined to this set of error codes.  There is however a "default" block there to handle a scenario whereby an unexpected Checklist error is added to the API but not to this list, which would result in it printing out the unmapped API error code with no mapping e.g. "AnotherNewValidationWarning".

I could remove the default block which would force us to explicitly handle just these error codes, but the risk is then that the above scenario wouldn't be handled.